share/completions/attributes.fish

```diff
@@ -0,0 +1,40 @@
+function __attributes_disk_generate_args
+  if not __fish_seen_subcommand_from set clear
```

Comment:
You can rewrite this outside the function as:

``fish
complete -c attributes  --no-files -n 'not __fish_seen_subcommand_from set clear' -a 'set' -d 'Set attribute'
complete -c attributes  --no-files -n 'not __fish_seen_subcommand_from set clear' -a 'clear' -d 'Clear attribute'
```